### ISSUE TITLE
[Question] convertig UUId on serverstart

### ISSUE THREAD

**Issue by CloudeLecaw** [FabioZumbi12/RedProtect#663]
Each start of our server we a getting this information inside of the log

```
[13:17:45 INFO]: [RedProtect] Updated region FoxyFelix, player 780258ef-1b16-3838-874e-27757f91a6a0 to 780258ef-1b16-3838-874e-27757f91a6a0
[13:17:45 INFO]: [RedProtect] Updated region FoxyFelix, player 5b412ef8-471d-3541-8fc3-a667470484e7 to 5b412ef8-471d-3541-8fc3-a667470484e7
[13:17:45 INFO]: [RedProtect] Updated region Freizeitpark, player 0ce9dc8f-5879-36de-b8c7-2428dbca4508 to 0ce9dc8f-5879-36de-b8c7-2428dbca4508
[13:17:45 INFO]: [RedProtect] Updated region f35blightning, player 6ca612f0-c901-3a35-8d8e-69417e923b3d to 6ca612f0-c901-3a35-8d8e-69417e923b3d
```

Why these UUIDs get upgraded each serverstart?
is there something wrong with their uuids?

**Comment by FabioZumbi12** [comment]
Yeah, you need to searh for regions by this guy and fix manually

**Comment by CloudeLecaw** [comment]
ok what what i have to fix there? the uuids look simliar

**Comment by CloudeLecaw** [comment]
We did what you said and found the issues. the problem was some currpted names as members in these zones
instead of **780258ef-1b16-3838-874e-27757f91a6a0@The Playersname** it was listed as
**780258ef-1b16-3838-874e-27757f91a6a0@780258ef-1b16-3838-874e-27757f91a6a0**
We changed it. so its fixed now. We have no clue waht was causing this, maybe one build long time ago..we realy dont know. Thnaks for your support <3
